addons/web_unsplash/static/src/js/unsplash_image_widget.js

```diff
@@ -76,35 +56,29 @@ widgetsMedia.ImageWidget.include({
     /**
      * @override
      */
-    _save: function () {
-        if (!this._unsplash.query) {
-            return this._super.apply(this, arguments);
+    _save: async function () {
+        const _super = this._super;
+        if (!Object.keys(this._unsplash.selectedImages).length) {
+            const images = await this._rpc({
+                route: '/web_unsplash/attachment/add',
+                params: {
+                    unsplashurls: this._unsplash.selectedImages,
+                    res_model: this.options.res_model,
+                    res_id: this.options.res_id,
+                    query: this._unsplash.query,
+                }
```

Comment:
Missing trailing comma, can you add it? (was like that before but...)

Reply:
I'm touching the line either way so might as well